JoyboyCommunity/src/shared/components/Post.tsx

```diff
@@ -42,6 +133,24 @@ interface PostProps {
 export const Post: React.FC<PostProps> = (props) => {
   const {post, event, repostedEvent} = props;
   const navigation = useNavigation<MainStackNavigationProps>();
+
+  const [isLiked, setIsLiked] = useState(false);
+  const [likes, setLikes] = useState(2);
+
+  const scale = useSharedValue(1);
+  const textOpacity = useSharedValue(likes > 0 ? 1 : 0);
+  const iconPosition = useSharedValue(likes > 0 ? 0 : 6); // Adjusted to account for the gap
```

Comment:
`useSharedValue` usage should not include computed value, since this will use the data from the relayers.